sympy/core/exprtools.py

```diff
@@ -1299,7 +1299,9 @@ def factor_nc(expr):
     """Return the factored form of ``expr`` while handling non-commutative
     expressions.
 
-    **examples**
```

Comment:
Good catch.
I did some spot testing, and it seems that the standard way to annotate "Examples" is with ```========```, so it would be nice to have that in line 1303.